source/chapters/appendix/mixxx_controls.rst

```diff
@@ -2482,6 +2482,24 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 1.10.0
 
 
+.. mixxx:control:: [ChannelN],stem_count
+
+   The number of stem track available for the loaded file. `0` means the deck is not a STEM deck.
+
+   :range: 0, 2..4, read-only
```

Comment:
Is 1 not in range? 
```suggestion
   :range: integer, 0...4, read-only
```

Reply:
No, if a stem deck as only one stem, then this a traditional deck, with a single stereo pair :)